frontend/components/product-list/sections/FilterableProductsSection/FacetFilter.tsx

```diff
@@ -0,0 +1,63 @@
+import { getProductListPath } from '@helpers/product-list-helpers';
+import { ProductList, ProductListType } from '@models/product-list';
+import * as React from 'react';
+import { RefinementList } from './RefinementList';
+import { RefinementMenu } from './RefinementMenu';
+
+type FacetFilterProps = {
+   attribute: string;
+   productList: ProductList;
+};
+
+export function FacetFilter({ attribute, productList }: FacetFilterProps) {
+   const createItemTypeURL = React.useCallback(
+      (itemType: string) => {
+         return (
+            getProductListPath({
+               ...productList,
+               type: ProductListType.DeviceItemTypeParts,
+               itemType,
+            }) ?? '#'
```

Comment:
It seems like `getProductListPath()` always returns a `string`, so the `?? '#'` might be unnecessary?